.eslintignore

```diff
@@ -0,0 +1,2 @@
+docker/
+server/
```

Comment:
Why do we want to ignore these? Maybe add a code comment to explain?

Reply:
iirc Ofer suggested that we should ignore these files. What do you think?

Reply:
I'm not exactly sure but this might cause some issues since the index.js file is JavaScript while the eslint config is for TypeScript.

Reply:
Makes sense! Let's add a code comment explaining that, if possible.